src/components/Calendar/Calendar.tsx

```diff
@@ -19,21 +19,57 @@ export default observer(function Calendar() {
         <div 
             className="ReactTimeline__Calendar"
             onDoubleClick={onDoubleClick}
+            style={{flexGrow:1, flexBasis: `${(spaces.customSpaces && spaces.customSpaces.length > 0) ? '60px' : '30px'}`, flexShrink: '0'}}
+
         >
             {spaces.grid.primary.map((x, i) => {
                 const time = Math.round(spaces.internalPxToTime(x));
                 const display = spaces.displayPrimary(time);
 
                 return (
-                    <div
+                    <><div
                         key={i}
                         className="ReactTimeline__Calendar-date"
-                        style={{ left: `${x}px`, top: `10px` }}
+                        style={{ left: `${x}px`, height: "25px" }}
```

Comment:
Eventually I think we should make the height overridable via a css class instead of a style set in js.  But good for now.